site/pages/meet-our-team/team-slice/index.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+/* eslint-disable react/no-set-state */
```

Comment:
Why do we have this rule in place anyway? We don't have Redux or any higher level state mgmt library so it seems a bit excessive. You can turn it off in `.eslintrc` if there isn't a reason.